cmd/opm/index/export.go

```diff
@@ -0,0 +1,96 @@
+package index
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	"github.com/operator-framework/operator-registry/pkg/lib/indexer"
+)
+
+var exportLong = templates.LongDesc(`
```

Comment:
This is new command so I think we need a doc for this.

Reply:
where could we add a doc for this?

Reply:
looks like in here would be good https://github.com/operator-framework/operator-registry/pull/149/files#diff-04c6e90faac2675aa89e2176d2eec7d8R65

Reply:
Yes I thought Kevin's PR has been merged by now but it is not. We should merge it given it is just a doc PR. Then we can add the information from this PR into that doc. PR: https://github.com/operator-framework/operator-registry/pull/149

Reply:
The #149 PR has been approved and hopefully merged soon. Then, the doc for this command can be added to that doc.

Reply:
I updated the docs but I'd like to keep this long desc to increase visibility on the fact that we are deprecating appregistry.

Reply:
I think that is fine.